leetcode/src/1189.c

```diff
@@ -0,0 +1,48 @@
+int maxNumberOfBalloons(char * text){
+    /* 
+        0 -> b, 
+        1 -> a,
+        2 -> l,
+        3 -> l,
+        4 -> o,
+        5 -> o,
+        6 -> n
+    */
+    int count_letters[7] = {0};
+    
+    /* Count the number of each letter */
+    for(int i = 0; text[i] != '\0'; i++){
+        if(text[i] == 'b') {
```

Comment:
I think it is better to use *ptr for traversing the string

Reply:
With that modification the code seems more readable but just for curiosity does this make any difference in terms of optimization?

Reply:
Ah it does not have any optimization. It is just more readable, and safe.